### DISCUSSION TITLE
Feat: Theme List

### DISCUSSION THREAD

**Discussion by sethgrid** [ghostty-org/ghostty#4125] [Feature Requests, Ideas] [answered]
Feature request / idea: 
Allow the config for the theme to be a list. When config is applied (new window or cmd + shift + , ), use the next (or random) element in the list. 

This would add some fun and surprise when opening new windows.

**Comment by JakeWharton** [comment, marked as answer]
You already suggested this here: https://github.com/ghostty-org/ghostty/discussions/3577, no?

**Reply by sethgrid** [reply]
there it is! I was thinking I was crazy or that I had closed a tab prematurely. I tried searching and couldn't find it; I wonder what was wrong with my github-search-fu. Thanks for linking!
